recipes/libde265/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "libde265" %}
+{% set version = "1.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/strukturag/libde265/releases/download/v{{ version }}/libde265-{{ version }}.tar.gz
+  sha256: 29bc6b64bf658d81a4446a3f98e0e4636fd4fd3d971b072d440cef987d5439de
+
+build:
+  number: 0
+
```

```suggestion
build:
  number: 0
  run_exports:
    - {{ pin_subpackage('libde256', max_pin='x.x.x') }}

```
Run exports ensure that the correct version is exported to the run dependencies of packages built against this package. Please adjust the *API* max_pin for the *ABI* compatability of this library. If you don't know what at what versions the ABI will break, pinning to 'x.x.x' is sufficiently safe.

---

Didn't know about run_exports, it is pinned as suggested now.